satpy/tests/reader_tests/test_agri_l1.py

```diff
@@ -286,6 +292,11 @@ def test_fy4a_all_resolutions(self):
             else:
                 self.assertEqual('K', res[band_name].attrs['units'])
 
+        # check whether the data type of orbital_parameters is float
+        orbital_parameters = res[band_names[0]].attrs['orbital_parameters']
+        for attr in orbital_parameters:
+            self.assertEqual(type(orbital_parameters[attr]), float)
```

Comment:
This would be a nice opportunity to also test the contents of the `orbital_parameters` dictionary here ;)

Reply:
OK. I switched to check the exact values instead of type.

Reply:
I meant testing both type and value, sorry if I wasn't clear on that.

Reply:
Em ... If the type is `array`, then `assertEqual` would give `False`. So, this statement actually checks both type and value, right?

Reply:
I didn't check `assertEqual`, but at least this evaluates to `True`:

```python
In [11]: np.array(0) == 0.                                                                                                                                                                                                                   
Out[11]: True
```

Reply:
Oh, my fault! I tested it like:

```
a = [0.]
a == 0.
```

Forgot to add `np.array()` ...

Reply:
Oh, in `get_test_content` you should probably set 
```
'/attr/NOMCenterLat': np.array(0.0),
```
instead of 
```
'/attr/NOMCenterLat': np.array([0.0]),
```
to trigger the original bug

Reply:
Thanks, hope I solved all of them.